Packs/SecLyticsAugurIOCsFeed/Integrations/SeclyticsFeed/SeclyticsFeed.py

```diff
@@ -0,0 +1,230 @@
+        tlp_color: Traffic Light Protocol color.
+    Returns:
+        str, dict, list. the markdown table, context JSON and list of indicators
+    """
+    offset = int(args.get('offset', 0))
+    limit = int(args.get('limit', 1000))
+
+    indicators = fetch_indicators_command(client, feed_tags, tlp_color, limit, offset)
+
+    hr_indicators = []
+    for indicator in indicators:
+        hr_indicators.append({
+            'Value': indicator.get('value'),
+            'Type': indicator.get('type'),
+            'rawJSON': indicator.get('rawJSON'),
+            'fields': indicator.get('fields'),
+        })
+
+    human_readable = tableToMarkdown("Indicators from Seclytics:", hr_indicators,
+                                     headers=['Value', 'Type', 'rawJSON', 'fields'], removeNull=True)
+
+    if args.get('limit'):
+        human_readable = human_readable + f"\nTo bring the next batch of indicators " \
+                                          f"run:\n!Seclytics-daily-get-indicators " \
+                                          f"limit={args.get('limit')} " \
+                                          f"offset={int(str(args.get('limit'))) + int(str(args.get('offset')))}"
+
+    return human_readable, {}, indicators
+
+
+def fetch_indicators_command(client: Client, feed_tags: List, tlp_color: Optional[str], limit=None, offset=None) \
+        -> list:
+    """Fetch-indicators command from Seclytics Feeds
+
+    Args:
+        client(Client): Seclytics Feed client.
+        feed_tags: The indicator tags.
+        tlp_color: Traffic Light Protocol color.
+        limit: limit the amount of indicators fetched.
+        offset: the index of the first index to fetch.
+
+    Returns:
+        list. List of indicators.
+    """
+    indicators = client.build_iterator(feed_tags, tlp_color, limit, offset)
+
+    return indicators
+
+
+def main():
+    params = demisto.params()
+    feed_tags = argToList(params.get('feedTags'))
+    tlp_color = params.get('tlp_color')
+
+    command = demisto.command()
+    demisto.info(f'Command being called is {command}')
+
+    commands = {
+        'test-module': module_test_command,
+        'seclytics-get-indicators': get_indicators_command
+    }
+    try:
+        seclytics_key = params.get('api_key').strip()
+        client = Client(api_key=seclytics_key,
+                        insecure=params.get('insecure'))
+
+        if demisto.command() == 'fetch-indicators':
+            indicators = fetch_indicators_command(client, feed_tags, tlp_color)
+            # we submit the indicators in batches
+            for b in batch(indicators, batch_size=2000):
+                demisto.createIndicators(b)
+        else:
+            readable_output, outputs, raw_response = commands[command](client, demisto.args(),
+                                                                       feed_tags, tlp_color)  # type: ignore
+            return_outputs(readable_output, outputs, raw_response)
```

Comment:
as mentioned above, our convention is to using the `return_results` with an instance of `CommandResults` (or 'ok' in case of the `test_module`)
```suggestion
            results = commands[command](client, demisto.args(),
                                                                       feed_tags, tlp_color)  # type: ignore
            return_results(results)
```